src/styles/illustrated-message.scss

```diff
@@ -0,0 +1,166 @@
+@import 'mixins/mixins';
+
+$block: fd-illustrated-message;
+
+.#{$block} {
+  @include fd-reset();
+
+  @include fd-flex(column) {
+    align-items: center;
+  }
+
+  width: 100%;
+  max-width: 30rem;
+  height: auto;
+  padding: 1rem;
+  margin-top: 5rem;
+  text-align: center;
+
+  &__illustration {
+    @include fd-reset();
+
+    height: 15rem;
+    width: 20rem;
+  }
+
+  &__figcaption {
+    @include fd-reset();
+
```

Comment:
please remove the empty line between 2 includes

Reply:
https://github.com/SAP/fundamental-styles/pull/2917#discussion_r754076609

Reply:
hmm that's weird...
on lines 62 and 63 of the same file you have the same case and I've never had problems like this before

Reply:
The issue only appears when one of the at-rules has a block of code `{ ... }`. Do you think we need to change this?

Reply:
nah, leave it like this. We are moving to saas soon so we will do some clean up there. Looks good for now